code-challenges.test.js

```diff
@@ -20,19 +20,58 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+
+describe('function1', () => {
+    it ('removes the first item from the array and shuffles the remaining items.', () => {
+        expect(colors1).toEqual(expect.arrayContaining(function1(colors1)));
+        expect(colors2).toEqual(expect.arrayContaining(function1(colors2)));
+        expect(function1(colors1)).not.toEqual(expect.arrayContaining(["purple"]))
+        expect(function1(colors2)).not.toEqual(expect.arrayContaining(["chartreuse"]))
+    })
+})
+
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// input: an array 
+// output: the array with the first item removed and shuffled. 
+// create a function that takes in an array
+// use .slice method to return  all the items after the first element [1] from the OG array in the form of an array 
+//  then use .sort() function to sort the new returned array and set the comparison function  by using 0.5 - Math.random, and this give a range from -0.5 to 0.5. The sort function will sort the array in an ascending order, but the random values from -0.5 to 0.5 randomize the order.  
+
+const function1 = (array) => {
+    return array.slice(1).sort(function(){return 0.5 - Math.random()})
+}
```

Comment:
I LOVE THIS!  great code, excellent description of how to use the sort method to randomize.  Well done, Joseph!